pkgconfig/oapv.pc.in

```diff
@@ -7,7 +7,7 @@ includedir=@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_INCLUDEDIR@/@LIB_NAME_BASE@
 Name: oapv
 Description: Advanced Professional Video Codec
 
-Version: @PROJECT_VERSION_MAJOR@.@PROJECT_VERSION_MINOR@.@PROJECT_VERSION_PATCH@
+Version: @LIB_VERSION_MAJOR@.@LIB_VERSION_MINOR@.@LIB_VERSION_PATCH@
```

Comment:
No, this one is fine staying as project version. Pkg-config is about a package, not a specific library within it.